submarine-server/server-api/src/main/java/org/apache/submarine/server/api/job/JobId.java

```diff
@@ -43,6 +48,25 @@ public static JobId newInstance(long serverTimestamp, int id) {
     return jobId;
   }
 
+  /**
+   * Get the object of JobId.
+   * @param jobIdString job id string
+   * @return object
+   */
+  public static JobId newInstance(String jobIdString) {
```

Comment:
Change `jobIdString` to `id`?

Reply:
I changed the method name and keep the param name. I think the `jobIdString` is better to explain the argument type.

Reply:
The IDE already has very smart type hints.

Reply:
Yeah, I agreed. Maybe used `jobId` is better than `id`, because in `JobId` we owned `getId` method to get the count of jobs since the server started. Any thoughts?

Reply:
Okay.

Reply:
Done.